packages/integrations/src/captureconsole.ts

```diff
@@ -18,12 +18,12 @@ export class CaptureConsole implements Integration {
   /**
    * @inheritDoc
    */
-  private readonly _levels: typeof CONSOLE_LEVELS = CONSOLE_LEVELS;
+  private readonly _levels: readonly string[] = CONSOLE_LEVELS;
```

Comment:
```suggestion
  private readonly _levels: typeof CONSOLE_LEVELS[number] = CONSOLE_LEVELS;
```

Reply:
Thought about that one too but that would still break the API. It was `string[]` before and doing `typeof CONSOLE_LEVELS[number]` is stricter than that. We could do that in the major bump however.